vars/pipelineVars.groovy

```diff
@@ -37,7 +37,7 @@ class pipelineVars implements Serializable {
     )
     String iqeCoreImage = 'quay.io/cloudservices/iqe-core:latest'
     String iqeTestsImage = 'quay.io/cloudservices/iqe-tests:latest'
-    String seleniumImage = 'quay.io/redhatqe/selenium-standalone:ff_91.5.1esr_gecko_v0.30.0_chrome_98.0.4758.80'
+    String seleniumImage = 'quay.io/redhatqe/selenium-standalone:ff_91.8.0esr_chrome_101.0.4951.41'
```

Comment:
We are removing `gecko` from the image - how does the selenium tags naming format works ?

Reply:
I decided that it doesn't make sense to include this information in the tag. I did in the past because I built images for the latest and prior gecko versions. Now I just build the images only with the latest gecko.